doc/source/ray-contribute/development.rst

```diff
@@ -131,7 +131,7 @@ To build Ray on Ubuntu, run the following commands:
   sudo apt-get install -y build-essential curl clang-12 pkg-config psmisc unzip
 
   # Install Bazelisk.
-  ci/env/install-bazel.sh
+  ray/cienv/install-bazel.sh
```

Comment:
hmm, i think `ray/cienv` is not a valid path, but let me know otherwise

`ci/env` is a valid path under ray checkout repository

Reply:
Thanks for your review and I am sorry for bothering you with the careless mistake.
I have pushed valid code.